content/en/core/overview/data-flows-for-analytics/cht-sync.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "CHT-Sync and CHT-Pipeleine setup"
```

Comment:
For consistency we use lowercase and hyphen separated names when talking about the repo, but caps and space separated in English titles.

```suggestion
title: "CHT Sync and CHT Pipeline setup"
```